### ISSUE TITLE
Import from external source not working anymore

### ISSUE THREAD

**Issue by ImAWerewolfDuck** [avluis/Hentoid#1031]
**Hentoid Version**:
1.17.2 (602)

**Device/Android Version**:
Oneplus 7 Pro, Android 11

**Issue details / Repro steps**:
When trying to import something the import stops.
To reproduce this I just need to try to import something (cbz, cbr or folder does not matter).
I also tried changing the folder name in which the files are saved and tried to delete the .json files but to no avail.
Now the json files are not being generated anymore.
Last time I updated my collection (~6months ago) it worked perfectly fine.
I shortened the log (it is always the same error message anyway) as it would be quite long otherwise.

**Stack trace / LogCat**:
```ruby
Import external log : begin
Hentoid ver: 1.17.2 (602)
API: 30
Device: GM1913
Start : 2023-01-02T18:34:36.303Z
End : 2023-01-02T18:37:09.167Z (02:32)
-----
Import books starting - initial detected count : 6516
Import book KO! (book already in collection) : content://com.android.externalstorage.documents/tree/primary%3Ags/document/primary%3Ags%2F%5BThomas%5D%20Who%20Does%20the%20Offline%20Meetup%20Princess%20Belong%20To___%20(Comic%20Kairakuten%20BEAST%202022-06).cbz
Import book KO! (book already in collection) : content://com.android.externalstorage.documents/tree/primary%3Ags/document/primary%3Ags%2F%5BShinozuka%20George%5D%20Let's%20Play%20Doubles!%20(Comic%20Shitsurakuten%202016-04).cbz

Import books complete - 0 OK; 6516 KO; 6516 final count
Import external log : end
```

@AVnetWS/admin-team

**Comment by RobbWatershed** [comment]
Thanks for your report.

The issue has been fixed. The fix will be available on next update.

**Comment by RobbWatershed** [comment]
Fixed on today's v1.17.3
